spec/game_spec.rb

```diff
@@ -8,6 +8,11 @@
   let(:game) { Game.new }
   let(:line) { '2:22 Kill: 3 2 10: Isgalamido killed Dono da Bola by MOD_RAILGUN' }
 
+  describe '#initialize' do
+    it 'creates a new game instance' do
+      expect(Game.new).to be_an_instance_of(Game)
```

Comment:
You already did it on let `:game`. Just use it.

Reply:
Also, to check if `expect(Anything.new).to be_an_instance_of(Anything)` isn't testing your code. You're testing if Ruby can create instances. That spec will not help you at all.